README.md

```diff
@@ -3,6 +3,7 @@
 This repository holds sample code for a number of data structures implemented with python.
 
 20150204 - Linked list add
+20150205 - Stack add
```

Comment:
I want better descriptions of what this all does.  Write me up some documentation.